ReactApp/src/components/Editor.jsx

```diff
@@ -53,8 +53,8 @@ const POST_MOCK = {
 
 function Editor({ post = POST_MOCK }) {
 	const [blocks, setBlocks] = useState([]);
-	const [registeredBlocks, setRegisteredBlocks] = useState([]);
-	const [isCodeEditorEnabled, setCodeEditorEnabled] = useState(false);
+	const [registeredBlocks] = useState([]);
+	const [setCodeEditorEnabled] = useState(false);
```

Comment:
I interpret the unused values were removed to address linter warnings. I suggest we prefix them with `_` instead of removing them, as removing `isCodeEditorEnabled` modifies the code behavior. 

The [`useState`](https://react.dev/reference/react/useState#reference) React Hook returns a positional array. Removing the `isCodeEditorEnabled` from this destructured array effectively renames the first item (`Boolean`) to `setCodeEditorEnabled` (implied `Function`). 

```suggestion
	const [registeredBlocks] = useState([]);
	const [_isCodeEditorEnabled, setCodeEditorEnabled] = useState(false);
```